kmir/src/kmir/kmir.py

```diff
@@ -113,6 +114,7 @@ def run(program_file: Path) -> CompletedProcess:
                 pipe_stderr=True,
                 pmap={'PGM': str(self.mir_parser)},
                 bug_report=self.bug_report,
+                depth=depth,
```

Comment:
@Baltoli Do you know why a call to `_krun` with a depth that interrupts the process would end in
```
[Error] krun: 
/home/daniel/.kbuild/mir-semantics/5a0ac5c/target/v6.0.24/llvm/interpreter 
/tmp/.krun-2023-08-03-09-04-52-Z4PGRkFugW/tmp.in.tUEaHwl7nA 2 
/tmp/.krun-2023-08-03-09-04-52-Z4PGRkFugW/result.kore

Command krun exited with code 4 for: /tmp/tmpkao_an5u
```

The `configuration` at the correct depth is printed beforehand as expected, but `_krun` then errors. This can be reproduced from `mir-semantics/kmir` with 
```
poetry run kmir run --definition-dir $(poetry run kbuild which llvm) --max-depth=2 --output pretty src/tests/integration/test-data/handwritten-rust/test-binop.mir
```